superset-frontend/src/views/components/Menu.tsx

```diff
@@ -172,18 +177,34 @@ const StyledHeader = styled.header`
     }
   }
 `;
-
+const globalStyles = (theme: SupersetTheme) => css`
+  .ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light.ant-menu-submenu-placement-bottomLeft {
+    border-radius: 0px;
+  }
+  .ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light {
+    border-radius: 0px;
+  }
+  .ant-menu-vertical > .ant-menu-submenu.data-menu > .ant-menu-submenu-title {
+    height: 28px;
+    i {
+      padding-right: ${theme.gridUnit * 2}px;
+      margin-left: ${theme.gridUnit * 1.75}px;
+    }
+  }
+`;
 const { SubMenu } = DropdownMenu;
 
 const { useBreakpoint } = Grid;
 
 export function Menu({
-  data: { menu, brand, navbar_right: navbarRight, settings },
+  data: { menu, brand, navbar_right: navbarRight, settings, allowedExtensions },
   isFrontendRoute = () => false,
 }: MenuProps) {
   const [showMenu, setMenu] = useState<MenuMode>('horizontal');
   const screens = useBreakpoint();
   const uiConig = useUiConfig();
+  const theme = useTheme();
+  const filteredMenu = menu.filter(item => item.name !== 'Upload Data');
```

Comment:
It doesn't look like we have this value `Upload Data` any more on the backend. Can we remove this filter?

Reply:
yep!